godump.go

```diff
@@ -176,6 +177,36 @@ func (d *Dumper) DumpStr(vs ...any) string {
 	return sb.String()
 }
 
+// DumpJSONStr pretty-prints values as JSON and returns it as a string.
+func (d *Dumper) DumpJSONStr(vs ...any) string {
+	if len(vs) == 0 {
+		return `{"error": "DumpJSON called with no arguments"}`
+	}
+
+	var data any = vs
+	if len(vs) == 1 {
+		data = vs[0]
+	}
+
+	b, err := json.MarshalIndent(data, "", strings.Repeat(" ", indentWidth))
+	if err != nil {
+		// Return valid JSON error response
+		errorJSON, marshalErr := json.Marshal(map[string]string{"error": err.Error()})
+		if marshalErr != nil {
+			// Fallback to raw JSON string if even that fails
+			return fmt.Sprintf(`{"error": "%q"}`, err.Error())
```

Comment:
```suggestion
			return fmt.Sprintf(`{"error": %q}`, err.Error())
```

Otherwise you will have 4 quotes